<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Userinfo</class>
 <widget class="QDialog" name="Userinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informações de Usuário</string>
  </property>
  <widget class="QLabel" name="label_db">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>691</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_funcao">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_limpar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuário</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>20</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>50</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Fone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fone">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>50</y>
     <width>33</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
